src/components/dialogs/network-modifications/two-windings-transformer/tap-changer-pane/phase-tap-changer-pane/phase-tap-changer-pane.js

```diff
@@ -135,7 +135,8 @@ const PhaseTapChangerPane = ({
             phaseTapChangerEnabledWatch &&
             (regulationMode === PHASE_REGULATION_MODES.CURRENT_LIMITER.id ||
                 regulationMode ===
-                    PHASE_REGULATION_MODES.ACTIVE_POWER_CONTROL.id)
+                    PHASE_REGULATION_MODES.ACTIVE_POWER_CONTROL.id ||
+                regulationMode === PHASE_REGULATION_MODES.FIXED_TAP.id)
```

Comment:
remove this useMemo, it has become useless since it is always true, and replace it with "phaseTapChangerEnabledWatch" in "disabled" prop for sideField, regulationTypeField, currentLimiterRegulatingValueField, flowSetPointRegulatingValueField, targetDeadbandField and regulatingTerminalField

Reply:
done